### PR TITLE
CI Integration

### LINKED ISSUE #3: Setup CI
CI builds should:

- Run tests
- ?

### PR DESCRIPTION
**Pull request by alukach**
This PR:

1. Adds CI support (closes #3):
    1. runs tests
    1. builds the project 
    1. generates documentation on every push
    1. (on pushes to `main`) deploys the Jekyll documentation to the `gh-pages` branch
1. Finalizes `_config.yaml`
1. Adds github logo to github link
1. Removes development helper comments

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+
+      - name: Setup node.js 🏡
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.15.0'
```
Comment (markdboyd) [inline .github/workflows/deploy.yaml:13]: If we're going to test against Node 14.15, let's add an `.nvmrc` to use that locally

File: packages/hello-world/__tests__/hello-world.test.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+"use strict";
```
Comment (markdboyd) [inline packages/hello-world/__tests__/hello-world.test.ts:1]: Test files in TS? Oh my

File: .github/workflows/deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Install 🔧
+        run: |
+          npm install
+          npm run bootstrap
+
+      - name: Test 🧪
+        run: |
+          npm test
+
+      - name: Build 🏗️
```
Comment (markdboyd) [inline .github/workflows/deploy.yaml:32]: We don't need to build before we test because we're testing against the TS directly?

**Review by markdboyd — CHANGES_REQUESTED** [review]
Looks great! I have one very small requested change

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (alukach) [reply .github/workflows/deploy.yaml:13]: This value actually came from our `.nvmrc` (that you added in b3613d1 😉 )
https://github.com/developmentseed/cdk-seed/blob/fdff42f5d5284ca7a30f343f691efa4995e06a9f/.nvmrc#L1

Comment (markdboyd) [reply .github/workflows/deploy.yaml:13]: Well then, touché

File: .github/workflows/deploy.yaml (hunk 3 above)
Comment (alukach) [reply .github/workflows/deploy.yaml:32]: Right, this PR adds a transform to Jest that allows Jest to do the translating on the fly.

https://github.com/developmentseed/cdk-seed/blob/0b2115115302f11941d636094c08c5e8f36668db/package.json#L28-L31

**Review by markdboyd — APPROVED** [review]

File: packages/hello-world/__tests__/hello-world.test.ts (hunk 2 above)
Comment (alukach) [reply packages/hello-world/__tests__/hello-world.test.ts:1]: 💥 

We don't actually need to do this, however I find the imports and type-completion to be a bit simpler.

**Pull request merged**